src/utils/database.ts

```diff
@@ -0,0 +1,40 @@
+import { existsSync, readFileSync, writeFileSync, unlinkSync } from 'fs';
+
+export class Database {
+  defaultFilePath: string;
```

Comment:
You can assign the default value here. Typescript also supports scoping like "private" "public" "protected". Should this be private?